### DISCUSSION TITLE
Problem when install the shadcn

### DISCUSSION THREAD

**Discussion by Gautam5514** [community/community#142758] [npm] [answered]
npx shadcn@latest init
If i do that then the problem has occur when it install the at the end the message will send 
:\NextJs\SaaS-Application\imaginify>npx shadcn@latest init
√ Preflight checks.
√ Verifying framework. Found Next.js.
√ Validating Tailwind CSS.
√ Validating import alias.
√ Which style would you like to use? » New York
√ Which color would you like to use as the base color? » Neutral
√ Would you like to use CSS variables for theming? ... no / yes
√ Writing components.json.
| Checking registry.

Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.

request to https://ui.shadcn.com/r/styles/new-york/utils.json failed, reason: read ECONNRESET
this problem has occur
![Screenshot 2024-10-27 123947](https://github.com/user-attachments/assets/53308bf5-d853-440f-afad-a0f9149afb64)

# If Install shadcn in vs code then this error will occur
please check this and give the proper solution on that
<img width="889" alt="image" src="https://github.com/user-attachments/assets/078cad07-06c8-4023-9aed-3fbf66a30318">

And one more problem will occur when use first time install the next js then the error will occur so please check this and please support me what is the problem has occur

![Screenshot 2024-10-26 201335](https://github.com/user-attachments/assets/0b159834-afc6-41bd-8090-481885c7f681)

**Comment by thomassiard** [comment]
The ECONNRESET error typically happens when a network connection is unexpectedly closed, possibly due to a network issue or the server being temporarily unavailable. Here are some steps you can try to resolve the issue:

Check Your Network Connection: Ensure that you have a stable internet connection. You might also try switching to a different network or disabling VPN/proxy if you’re using one, as they can sometimes interfere with requests.

Retry the Command: The server might have been temporarily down when you ran the command. Try running npx shadcn@latest init again after a few minutes to see if it goes through.

Clear npx Cache: Sometimes, cached packages can cause issues. Try clearing the cache with:

bash
Copy code
npx clear-npx-cache
Then, run the command again.

Use a Different Package Manager: If the issue persists, you can try using yarn instead of npx to see if it bypasses the network issue:

bash
Copy code
yarn create shadcn@latest init
Check GitHub for Open Issues: If the problem persists, it may be a known issue. You can check the [shadcn GitHub repository](https://github.com/shadcn/ui/issues) for similar issues or open a new one if there isn’t an existing solution.

**Comment by Smnthjm08** [comment, marked as answer]
add this for the html tag in the root layout  `<html lang="en" suppressHydrationWarning>` at the opening of the html tag . This should fix it.
